requirements.txt

```diff
@@ -0,0 +1 @@
+pip install AIF360
```

Comment:
Please read [this page](https://pip.pypa.io/en/stable/user_guide/#requirements-files), and the links within for directions on how requirements.txt files should be formatted.  I would also add pandas into your file since that is not always included in basic python distributions.